runtime/compiler/net/ClientStream.hpp

```diff
@@ -88,13 +87,13 @@ class ClientStream : public CommunicationStream
       {
       if (getVersionCheckStatus() == NOT_DONE)
          {
-         _cMsg.set_version(getJITServerVersion());
+         _cMsg.setVersion(getJITServerVersion());
          write(MessageType::compilationRequest, args...);
-         _cMsg.clear_version();
+         _cMsg.clearVersion();
          }
       else // getVersionCheckStatus() == PASSED
          {
-         _cMsg.clear_version(); // the compatibility check is done. We clear the version to save message size.
+         _cMsg.clearVersion(); // the compatibility check is done. We clear the version to save message size.
```

Comment:
Right, I didn't actually read what this is for.

Reply:
Should I just remove version check status entirely?

Reply:
You can delete `clearVersion()` because it doesn't do anything useful. I'll have to think if we can eliminate the version from the messages. Anyway, to keep things simple I would leave this version check optimization for another PR.